frontend/packages/client/src/App.js

```diff
@@ -31,7 +32,8 @@ function App() {
       <QueryClientProvider client={queryClient}>
         <Web3Provider network={process.env.REACT_APP_FLOW_ENV}>
           <Router>
-            <ChakraProvider>
+            {/* using resetCSS to avoid conficts */}
+            <ChakraProvider theme={theme} resetCSS={false}>
```

Comment:
Looks like adding the provider as some overwrites to styles using resetCSS to prevent it @mannynotfound